bedrock/firefox/templates/firefox/developer/whatsnew.html

```diff
@@ -25,7 +25,11 @@
       {{ high_res_img('protocol/img/logos/firefox/browser/developer/logo-word-hor-md.png', {'alt': '', 'width': '368', 'height': '68'}) }}
 
       {% if ftl_has_messages('firefox-developer-congrats-you-now-have', 'firefox-developer-welcome-to-your-new-favorite') %}
-        <h1 class="intro-title">{{ ftl('firefox-developer-congrats-you-now-have') }}</h1>
+        {% if oldversion and ftl_has_messages('firefox-developer-congrats-you-now-have-latest') %}
```

Comment:
I think we can actually simplify this a bit and remove the `oldversion` check.

This /whatsnew page is only ever shown on a browser update. New installations see a different [/firstrun](https://www.mozilla.org/en-US/firefox/88.0a2/firstrun/) page when a profile is first created. So it's safe to assume that the new string you've added here is always applicable.

Does that make sense?

Reply:
Actually, I was not sure whether the /whatsnew page is only for update or for both update and installation. So I added the `old_version` check.  
As you clarified now, I am going to remove this check. But I have a question though.
@alexgibson is there any reason for keeping the `old_version` in the get_template_name method of WhatsnewView? Because old_version is used nowhere after the [declaration](https://github.com/mozilla/bedrock/blob/daa9b16a786fe7aaca9d543c08ea71a4aed583a0/bedrock/firefox/views.py#L630).